beacon-chain/state/stateutil/validator_root.go

```diff
@@ -87,3 +83,40 @@ func Uint64ListRootWithRegistryLimit(balances []uint64) ([32]byte, error) {
 	binary.LittleEndian.PutUint64(balancesLengthRoot, uint64(len(balances)))
 	return ssz.MixInLength(balancesRootsRoot, balancesLengthRoot), nil
 }
+
+// PackUint64IntoChunks packs a list of uint64 values into 32 byte roots.
+func PackUint64IntoChunks(vals []uint64) ([][32]byte, error) {
```

Comment:
Shouldn't this be called `PackBalancesIntoChunks`? You use balances in code and comments inside the function.

Reply:
This is applied for all uint64 lists not just for balances.